src/amp-story-player/README.md

```diff
@@ -0,0 +1,449 @@
+
+This update contains guidance on using the Web Story Player to support:
+
+- [Navigating between stories](#Navigating-between-stories)
+- [Integrating Web Stories into your website (entrypoints)](<#Integrating-web-stories-into-your-website-(entrypoints)>)
+- [(Beta) Programmatically fetching more stories](<#(Beta)-Programmatically-fetching-more-stories>)
+- [(Beta) Circular wrapping](<#(Beta)-Circular-wrapping>)
+- [(Beta) Customizing position & visibility of story UI controls](<#(Beta)-Customizing-position-&-visibility-of-story-UI-controls>)
+- [(Beta) Custom Events](<#(Beta)-Custom-Events>)
+- [Upcoming API Change: Poster Image](#Upcoming-API-Change:-Poster-Image)
+
+Beyond this, you’ll also find:
+
+- [Additional use case demos](#Additional-use-case-demos)
+- [Additional explorations and roadmap](#Additional-explorations-and-roadmap)
+- [FAQs](<#FAQs-(please-feel-free-to-reach-out-with-other-questions-&-thoughts)>)
+
+## Player setup
+
+If you need a refresher on how to set up the amp-story-player, including what scripts to import and how to add stories, visit the [documentation](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md).
+
+## Navigating between stories
+
+Navigating between stories inside of the player works out of the box for mobile. Simply swipe horizontally to go from one story to the other. [Full screen demo](https://web-player-cards-carousel.web.app/examples/amp-story/player-full-screen.html).
+
+To navigate between stories on desktop, there are a couple of options:
+
+<table>
+  <tr>
+    <td>1. Opt-in to the “skip-next-button” using the “controls” API (more on that below on the “Customizing position & visibility of story UI controls” section).</td>
+    <td><img src="https://github.com/ampproject/amphtml/blob/master/src/amp-story-player/img/skip-next-desktop.png?raw=true"></td>
+  </tr>
+  <tr>
+    <td>2. Use the Javascript APIs to implement your own navigational buttons.  If you want to use the Javascript APIs, you can use the <a href="https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md#go">go() method</a> (see documentation for more details) as shown in the following example:</td>
+    <td>
+
+```javascript
+const player = document.querySelector("amp-story-player");
+let init = false;
+
+// Waits for when player API is ready to be used.
+player.addEventListener("ready", () => {
+  initializeNavigationalButtons();
+});
+
+// Synchronous check to avoid race conditions where event was dispatched before // we started listening.
+if (player.isReady) {
+    initializeNavigationalButtons();
+}
+
+function initializeNavigationalButtons() {
+  if (init) return;
+  init = true;
```

Comment:
Nope, but since we are calling `initializeNavigationalButtons()` from two different places (event listener and synch property) it's nice to safeguard in case it one gets called after the other.

Reply:
It shouldn't be possible to have both called since we set isReady and dispatch "ready" at the same time, right?

Reply:
Yep good point.